make.py

```diff
@@ -15,6 +15,29 @@
 from pathlib import Path
 from typing import Iterable, Union
 
+CYAN = "\x1b[36m"
+GREY = "\x1b[37m"
+PINK = "\x1b[35m"
+BOLD_RED = "\x1b[1;31m"
+BOLD_YELLOW = "\x1b[1;33m"
+NO_COLOR = "\x1b[0;0m"
+CUTENESS = [
+    "ฅ^◕ﻌ◕^ฅ",
+    "(^･㉨･^)∫",
+    "^•ﻌ•^ฅ",
+    "✺◟(⏓ᴥ⏓▽)◞✺",
+    "ฅ(・㉨・˶)ฅ",
+    "(ﾐꆤ ﻌ ꆤﾐ)∫",
+    "(ﾐㆁ㉨ㆁﾐ)",
+    "ฅ(=ච ω ච=)",
+]
+
+
+class LazyStr(str):
+    def __str__(self):
+        return self
```

Comment:
It don't see the class used, it's expected ?

Reply:
removed.